### DISCUSSION TITLE
how to  generate shellcode as cobaltstrike stageless

### DISCUSSION THREAD

**Discussion by teamdArk5** [BishopFox/sliver#1607] [Q&A]
i has see the page https://sliver.sh/docs?name=Stagers,“As payloads can be pretty big (around 10MB),”  

so  how can i generate shellcode as  cobaltstrike stageless raw?

**Comment by rkervella** [comment]
Stageless sliver payloads are the ones that are 10+ MB, because they're just a sliver executable wrapped with the donut loader.
